### PR TITLE
UNIT TEST COVERAGE: cover the service command with unit tests

### PR DESCRIPTION
**Pull request by coderimus**
### Description (*)
Cover with Unit Tests the files command

### Fixed Issues (if relevant)
#811 

### Manual testing scenarios (*)
N/A

### CODE REVIEW

File: AdobeStockClient/Model/Client/Files.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 use Magento\Framework\Locale\ResolverInterface as LocaleResolver;
 use Magento\Framework\Serialize\Serializer\Json;
 use Magento\AdobeStockClientApi\Api\Client\FilesInterface;
-use Magento\Framework\Webapi\Exception as WebapiException;
+use Magento\Framework\Webapi\Exception as WebApiException;
```
Comment (sivaschenko) [inline AdobeStockClient/Model/Client/Files.php:19]: ok..

File: AdobeStockClient/Test/Unit/Model/Client/Files/CurlRequestFailed.php [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            ->method('getApiKey')
+            ->willReturn('string');
+
+        $this->getAccessTokenMock->expects($this->once())
+            ->method('execute')
+            ->willReturn('array');
+
+        $this->curlAdapterMock->expects($this->once())
+            ->method('setHeaders')
+            ->willReturn(null);
```
Comment (sivaschenko) [inline AdobeStockClient/Test/Unit/Model/Client/Files/CurlRequestFailed.php]: No need to declare returning `null` explicitly

File: AdobeStockClient/Test/Unit/Model/Client/Files/CurlResultSuccessFilesExistsTest.php [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        $this->imsConfigMock->expects($this->once())
+            ->method('getApiKey')
+            ->willReturn('string');
+
+        $this->getAccessTokenMock->expects($this->once())
+            ->method('execute')
+            ->willReturn('array');
+
+        $this->curlAdapterMock->expects($this->once())
+            ->method('setHeaders')
```
Comment (sivaschenko) [inline AdobeStockClient/Test/Unit/Model/Client/Files/CurlResultSuccessFilesExistsTest.php:127]: I believe `->with(...)` on `setHeaders` method is one of the most important assertions in the test

File: AdobeStockClient/Test/Unit/Model/Client/Files/CurlResultContainsEmptyFilesTest.php [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+     */
+    public function testResultContainsEmptyFiles(): void
+    {
+        $ids = [1];
+        $columns = [];
+        $locale = 'en';
+
+        $this->clientConfigMock->expects($this->once())
+            ->method('getProductName')
+            ->willReturn('string');
```
Comment (sivaschenko) [inline AdobeStockClient/Test/Unit/Model/Client/Files/CurlResultContainsEmptyFilesTest.php]: It's better to keep return values close to their actual possible value - as it helps to read the tests/test results

File: AdobeStockClient/Test/Unit/Model/Client/Files/CurlResultContainsEmptyFilesTest.php [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+        $this->imsConfigMock->expects($this->once())
+            ->method('getApiKey')
+            ->willReturn('string');
```
Comment (sivaschenko) [inline AdobeStockClient/Test/Unit/Model/Client/Files/CurlResultContainsEmptyFilesTest.php]: Values used in unit tests should be assigned to variables. There are 2 possible situations:
1. Value is used verification of value flow - should be stored in a variable and used at least 2 times
2. Value is used as an argument to a method call - it is a good idea to provide such values from data provider as they can be used for boundary testing

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @coderimus thanks for the pull request! Please see my review comments

**Comment by coderimus** [comment]
@sivaschenko thank you for your detail review. Requested and suggested changes done. Please, check when you will have time.

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**